pyiges/iges.py

```diff
@@ -1,13 +1,10 @@
 from tqdm import tqdm
+
+from pyiges.check_imports import assert_full_module_variant, pyvista, vtkAppendPolyData
+
 from pyiges.entity import Entity
 from pyiges import geometry
 
-try: 
-    import pyvista 
-    from pyvista._vtk import vtkAppendPolyData
-except ImportError:
-    pass 
```

Comment:
I removed this, looks like leftovers of a try to accomplish what we now do rigorously